okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -60,6 +60,8 @@
     @VisibleForTesting
     static final int NUM_SCHEDULING_THREADS = 5;
 
+    private static final boolean DEFAULT_ENABLE_HTTP2 = true;
```

Comment:
I think all of the other default values are added by the `ClientConfigurations.of` method - any reason to do this stuff inside the client here?